README.md

```diff
@@ -394,6 +394,79 @@ This creates a Python entrypoint, YAML configs and a Dockerfile. You can start f
 
 </details>
 
+#### MultimodalDocument
+  
+ A MultimodalDocument in Jina is a document composed by more than one chunks with different modalities.
+ 
+There are different ways to build your multimodal document: 
+ 
+##### From content modality mapping
+Build a Multimodal document by providing the modality names and the content of the chunks.
+  
+```python
+    from jina.types.document.multimodal import MultimodalDocument
+    image_title = 'my holiday picture'
+    image_description = 'the family having fun on the beach'
+    image = PIL.Image.open('path/to/image.jpg')
+    content_mapping = {
+        'title': image_title,
+        'description': image_description,
+        'image': image
+    }   
+    document = MultimodalDocument(modality_content_mapping=content_mapping)
+```
+
+##### From chunks with different modalities
+Build a Multimodal document by creating chunks. Useful when extra metadata needs to exist in every chunk
+  
+```python
+    from jina.types import Document
+    from jina.types.document.multimodal import MultimodalDocument
+    chunk_title = Document('my holiday picture')
+    chunk_title.modality = 'title'
+    
+    chunk_description = Document('the family having fun on the beach')
+    chunk_description.modality = 'description'
+
+    chunk_image = Document(Image.open('path/to/image.jpg'))
```

Comment:
I think the "From content modality mapping" and "From chunks with different modalities" are not necessary. To reduce the number of words, I wonder if we could just say:

We can build a multimodal document by providing the modality names and the content of the chunks:

`Code`

We can also create chunks when we have extra metadata in each chunk:

`Code`

Reply:
Use Title Case for headings: https://titlecase.com/